### ISSUE TITLE
Alexa.Speak Italiano

### ISSUE THREAD

**Issue by candrea77** [thorsten-gehrig/alexa-remote-control#21]
Hello to all,
Alexa is available in Italy from 30/10/2018 and I'm using an echo dot v3 with italian language.

I'm trying to use alexa-remote-control with the TTS function.
 My setup is for Italian WebSite :
LANGUAGE="it-IT"
AMAZON='amazon.it'
ALEXA='alexa.amazon.it'

If I send the command : -e speak:"che tempo farà domani" Alexa don't do TTS using this phrase, but read the phrase as a command , and then tell me the forecast (che tempo farà domani --> What's the weather like tomorrow ?)

Please note : if I set up the echo with English language, the TTS function works fine.

How to solve this issue ? It is an Alexa related problem, or the problem is with alexa-remote-control?

Thanks in advance

**Comment by Apollon77** [comment]
hey, it has a feature all other languages want to have ;-)

What you can try is to create a "speak" in the alexa App in Routines and see what happends there. The same or will it speak what you typed?

**Comment by Apollon77** [comment]
Interesting. In Germany and English you can also enter custom free text to speak. As Alternative to choose from a list.

**Comment by candrea77** [comment]
So it seems a different implementation ....

A question : alexa.speak is working in Ducth/German?

Regards

Il giorno Gio 1 Nov 2018, 20:09 Ingo Fischer <notifications@github.com> ha
scritto:

**Comment by manutremo** [comment]
Same issue in spanish Alexa as italian - it doesn't just speak the text, it tries to convert into an order. -e speak:'Buenos dias' just returns (in spanish) "Sorry I can't find an answer to your question".

**Comment by candrea77** [comment]
Hello manutremo,
can you confirm that also in spanish using Alexa app you cannot create a routine and specify a custom text for TTS with Alexa ?

**Comment by manutremo** [comment]
Yes I confirm that spanish alexa behaves the same as described above for italian.

**Comment by Apollon77** [comment]
Then, to be honest: someone should get a proxy into the app communication very fast to see if they maybe send something different on the message level in the app ... but I fear (because of the above info) that it is somehow "only" a bug on their server components.

BTW: You can fix that for you by using the equivalent to "Simon says ..." to your language

**Comment by Apollon77** [comment]
PS: was spanish also new since 31.10.?

**Comment by manutremo** [comment]
Yes, alexa spanish was launched on Oct30th.

The trick "Simon says" does not seem to work at least in the form "Simon dice"... I searched a list of available commands and I found "Alexa repite conmigo" (Alexa repeat with me), but it just returns "I can't repeat yet, but I'm learning"... which look like a hint that it's still being developed for this language.

**Comment by Apollon77** [comment]
But then this may be the reason!
Normally speak commands are treated as "Alexa replease with me" kind of commands internally with the entered text added. So if "repeat with me" is not available in your country then they may be the "Fallback" reason" to just do what was supposed to be said

**Comment by manutremo** [comment]
It looks like so... we'll need to wait until this response is finally deployed on new languages. I'll keep trying.

**Comment by manutremo** [comment]
Spanish seems to be already working... Just so you know, maybe Italian is too.

**Comment by candrea77** [comment]
Italian too

Il giorno Dom 2 Dic 2018, 17:02 manutremo <notifications@github.com> ha
scritto:
